docs/guides/migration_mechanism.md

```diff
@@ -88,24 +101,15 @@ Essentially, via the Firefly wallet, token holders:
 
 ### Verifier Node
 
-A verifier node was a Chrysalis Phase 2 node which upon seeing receipts:
-
-1. Queried a legacy node for the confirmation data for the specified milestone in the receipt.
-1. Then performed WOTS signature verification of the legacy milestone bundle and all confirmed bundles.
-1. Additionally it also checked that all confirmed funds on the legacy network for the given legacy milestone, were
-1. Produce migration bundles which sent funds to `migration addresses` controlled by the given owner.
-1. These bundles are confirmed on the legacy network.
-1. The Chrysalis Phase 2 Coordinator picks these confirmed legacy bundles up and generates receipts minting those funds
-   to the target Ed25519 address.
-1. Verifier nodes verify the receipts and make sure that the funds originate from the legacy network.
-
-### Verifier Node
-
 A verifier node is a Chrysalis Phase 2 node which upon seeing receipts:
 
 1. Queries a legacy node for the confirmation data for the specified milestone in the receipt.
-1. Then performs WOTS signature verification of the legacy milestone bundle and all confirmed bundles.
-1. Additionally it also checks that all confirmed funds on the legacy network for the given legacy milestone, are
-   indeed minted with a given batch of receipts (i.e. nothing is left out).
+2. Then performs WOTS signature verification of the legacy milestone bundle and all confirmed bundles.
+3. Additionally, it also checks that all confirmed funds on the legacy network for the given legacy milestone, are indeed minted with a given batch of receipts (i.e. nothing is left out).
+4. Produces migration bundles which send funds to `migration addresses` controlled by the given owner.
+5. The legacy networks confirms these bundles.
+6. The Chrysalis Phase 2 Coordinator picks these confirmed legacy bundles up and generates receipts minting those funds to the target Ed25519 address.
+7. Verifier nodes verify the receipts and make sure that the funds originate from the legacy network.
+
```

Comment:
This listing is messed up and doesn't make sense. Somehow explanations between what a verifier node does and how the migration works got mangled together.

This what it should entail:
![image](https://user-images.githubusercontent.com/1385611/133388678-8689ec3a-75e9-4afb-98a6-926a23dcf847.png)